package.json

```diff
@@ -30,8 +30,8 @@
     "build:es:watch": "NODE_ENV=development lerna run --no-private --parallel build:es:watch"
   },
   "workspaces": [
+    "application-templates/*",
```

Comment:
It is still a pretty bold assumption of ours that we will ever have more. But it would be awesome 😁

Reply:
I plan to 😜